host.go

```diff
@@ -221,18 +218,14 @@ func (h *Host) Stop() error {
 	return nil
 }
 
-// Advertise advertises in the DHT.
-func (h *Host) Advertise(strs []string) {
-	h.discovery.advertiseCh <- strs
+// RefreshNamespaces advertises in the DHT.
+func (h *Host) RefreshNamespaces() {
+	h.discovery.advertiseCh <- struct{}{}
```

Comment:
shouldn't this be called `Advertise`? how does it refresh namespaces?

Reply:
`Advertise` is actually what this function does, can you change the name back?

Reply:
Done